test/cmdlineTests/standard_eWasm_requested/output.json.orig

```diff
@@ -0,0 +1,354 @@
+    (param $x3 i64)
+    (param $x4 i64)
+    (result i64)
+    (local $v i64)
+    (block $label__1
+        (if (i64.ne (i64.extend_i32_u (i64.ne (i64.const 0) (i64.or (i64.or (local.get $x1) (local.get $x2)) (local.get $x3)))) (i64.const 0)) (then
+            (unreachable)))
+        (if (i64.ne (i64.extend_i32_u (i64.ne (i64.const 0) (i64.shr_u (local.get $x4) (i64.const 32)))) (i64.const 0)) (then
+            (unreachable)))
+        (local.set $v (i64.extend_i32_u (i32.wrap_i64 (local.get $x4))))
+
+    )
+    (local.get $v)
+)
+
+<<<<<<< HEAD
+(func $to_internal_i32ptr
+    (param $x1 i64)
+    (param $x2 i64)
+    (param $x3 i64)
+    (param $x4 i64)
+    (result i64)
+    (local $r i64)
+    (local $p i64)
+    (block $label__2
+        (local.set $p (call $u256_to_i32 (local.get $x1) (local.get $x2) (local.get $x3) (local.get $x4)))
+        (local.set $r (i64.extend_i32_u (i32.add (i32.wrap_i64 (local.get $p)) (i32.wrap_i64 (i64.const 64)))))
+        (if (i64.ne (i64.extend_i32_u (i32.lt_u (i32.wrap_i64 (local.get $r)) (i32.wrap_i64 (local.get $p)))) (i64.const 0)) (then
+            (unreachable)))
+
+    )
+    (local.get $r)
+)
+
+(func $codecopy
+=======
+(func $calldatacopy
+>>>>>>> cee41839f... Use calldatacopy for selector retrieval to avoid access out of bounds for calldata.
+    (param $x1 i64)
+    (param $x2 i64)
+    (param $x3 i64)
+    (param $x4 i64)
+    (param $y1 i64)
+    (param $y2 i64)
+    (param $y3 i64)
+    (param $y4 i64)
+    (param $z1 i64)
+    (param $z2 i64)
+    (param $z3 i64)
+    (param $z4 i64)
+<<<<<<< HEAD
+    (block $label__3
+        (call $eth.codeCopy (i32.wrap_i64 (call $to_internal_i32ptr (local.get $x1) (local.get $x2) (local.get $x3) (local.get $x4))) (i32.wrap_i64 (call $u256_to_i32 (local.get $y1) (local.get $y2) (local.get $y3) (local.get $y4))) (i32.wrap_i64 (call $u256_to_i32 (local.get $z1) (local.get $z2) (local.get $z3) (local.get $z4))))
+    )
+=======
+    (call $eth.callDataCopy (call $u256_to_i32 (local.get $x1) (local.get $x2) (local.get $x3) (local.get $x4)) (call $u256_to_i32 (local.get $y1) (local.get $y2) (local.get $y3) (local.get $y4)) (call $u256_to_i32 (local.get $z1) (local.get $z2) (local.get $z3) (local.get $z4)))
+>>>>>>> cee41839f... Use calldatacopy for selector retrieval to avoid access out of bounds for calldata.
+)
+
+(func $endian_swap_16
+    (param $x i64)
+    (result i64)
+    (local $y i64)
+    (block $label__4
+        (local.set $y (i64.or (i64.and (i64.shl (local.get $x) (i64.const 8)) (i64.const 65280)) (i64.and (i64.shr_u (local.get $x) (i64.const 8)) (i64.const 255))))
+
+    )
+    (local.get $y)
+)
+
+(func $endian_swap_32
+    (param $x i64)
+    (result i64)
+    (local $y i64)
+    (local $hi i64)
+    (block $label__5
+        (local.set $hi (i64.shl (call $endian_swap_16 (local.get $x)) (i64.const 16)))
+        (local.set $y (i64.or (local.get $hi) (call $endian_swap_16 (i64.shr_u (local.get $x) (i64.const 16)))))
+
+<<<<<<< HEAD
```

Comment:
conflict marker

Reply:
ah, looks like that I accidentally checked in that file.